modules/custom/az_dam/config/quickstart/core.entity_form_display.node.ib_media.default.yml

```diff
@@ -0,0 +1,87 @@
+uuid: 6be41c47-fe1b-4c11-88f2-76669d64953e
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.